skiko/src/awtMain/cpp/windows/directXRedrawer.cc

```diff
@@ -11,7 +11,7 @@
 #include "ganesh/GrDirectContext.h"
 #include "SkSurface.h"
 #include "include/gpu/ganesh/SkSurfaceGanesh.h"
-#include "../common/interop.hh"
+#include "../../../jvmMain/cpp/common/interop.hh"
```

Comment:
It doesn't look right. Things like `../../../jvmMain/` shouldn't be a part of sources

Could you please add `skiko/src/jvmMain/cpp/common` to header search paths instead?

Reply:
Done! I've squashed it with the first commit.